pkgs/applications/networking/feedreaders/castget/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+, glib, ronn, curl, id3lib, libxml2 }:
+
+stdenv.mkDerivation rec {
+  pname = "castget";
+  version = "1_2_4";
```

Comment:
I'd prefer having the version in the conventional point-separated style. For the `rev` you can then use `lib.replaceStrings ["."] ["-"] version`.

Reply:
Fixed.